kernel/src/arch/x86_64/mod.rs

```diff
@@ -689,6 +768,49 @@ fn _start(argc: isize, _argv: *const *const u8) -> isize {
         kcb.set_physical_memory_manager(tcache);
     }
 
+    // 1. Discover persistent memory using topology information.
```

Comment:
e.g., same question as above might be better if we can somehow make it part of bootlaoder?

Reply:
See above.